src/components/Button/Button.styled.ts

```diff
@@ -23,7 +23,7 @@ export const ButtonWrapper = styled.span`
   }
 
   & > a {
-	text-transform: uppercase;
+    text-transform: uppercase;
   }
 
   &.normalButton > button {

```

Comment:
I feel we should make the jump and update all "button" and "a" to .MuiButtonBase-root, that way we don't have to maintain 2 different styles for button links and regular buttons.